src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,11 @@
+package wolox.training.repositories;
+
+import java.util.List;
+import org.springframework.data.repository.Repository;
+import wolox.training.models.Book;
+
+public interface BookRepository extends Repository<Book, Long> {
+
+  List<Book> findByAuthor(String author);
```

Comment:
For this card, the method should return just one book. We can change the List<Book> for an Optional<Book> (so we can handle easier a null response) 😁

Reply:
Got it! I've pushed the changes :)